ckanext/datagovuk/controllers/user.py

```diff
@@ -8,9 +8,28 @@ def _new_form_to_db_schema(self):
         from ckanext.datagovuk.schema import user_new_form_schema
         return user_new_form_schema()
 
+    def _edit_form_to_db_schema(self):
+        from ckanext.datagovuk.schema import user_edit_form_schema
+        return user_edit_form_schema()
+
     def me(self, locale=None):
         if not c.user:
             h.redirect_to(locale=locale, controller='user', action='login',
                           id=None)
         user_ref = c.userobj.get_reference_preferred_for_uri()
         h.redirect_to(locale=locale, controller='user', action='dashboard_datasets')
+
+    def _get_form_password(self):
+        password1 = request.params.getone('password1')
+        password2 = request.params.getone('password2')
+        log.debug(password1)
```

Comment:
Hmm. We don't want to be logging passwords.

Reply:
Removed now.  Will have to make a new PR.